src/com/projectkorra/projectkorra/firebending/FireBlast.java

```diff
@@ -116,28 +140,25 @@ private void setFields() {
 		this.fireTicks = getConfig().getDouble("Abilities.Fire.FireBlast.FireTicks");
 		this.knockback = getConfig().getDouble("Abilities.Fire.FireBlast.Knockback");
 		this.flameRadius = getConfig().getDouble("Abilities.Fire.FireBlast.FlameParticleRadius");
-		this.smokeRadius = getConfig().getDouble("Abilities.Fire.FireBlast.SmokeParticleRadius");
 		this.random = new Random();
 	}
 
 	private void advanceLocation() {
 		if (this.isFireBurst) {
 			this.flameRadius += 0.06;
-			this.smokeRadius += 0.06;
 		}
-		
+
 		if (this.showParticles) {
-			ParticleEffect.FLAME.display(this.location, 6, this.flameRadius, this.flameRadius, this.flameRadius);
-			ParticleEffect.SMOKE_NORMAL.display(this.location, 3, this.smokeRadius, this.smokeRadius, this.smokeRadius);

```

Comment:
You are gonna want to keep this line. Removal of the flame particle in place of `playFirebendingParticles` is good but that doesn't play smoke.

Reply:
That was intentional, the smoke makes firebending look a lot less appealing and it sort of takes away from the look of the fire especially since we're trying to highlight the blue fire now so I decided it might be better to go without.

Reply:
Alright. If others approve then it's fine. It's just always been a thing.